### ISSUE TITLE
Send different values to API using hero card

### ISSUE THREAD

**Issue by rd-devvv** [microsoft/BotFramework-Composer#8954]
Hello,
 
I have a hero card that requests from user to select his employment status. my issue is how can I send different value to API?

For ex: in the hero card there is employed, self employed and unemployed etc... when user clicks on unemployed its storing the value unemployed in dialog.employmentstatus and all is working fine and I have a confirmation for user to make sure he clicked the right one, but when I want to send to API, the API accepts only es_unemployed or es_employed etc.. so when i send dialog.employment status to API its sending unemployed while I want the value to be sent as es_unemployed.

Thanks in advance,
Regards

![employmentstatus](https://user-images.githubusercontent.com/94966165/151944935-6a5214f6-f8c6-4b57-8227-ee51712a76e7.PNG)
.

**Comment by anishprasad01** [comment]
Thank you for the report. I will investigate.

**Comment by rd-devvv** [comment]
Hello @anishprasad01 ,
Any updates regarding the above?
Thank you for your help

**Comment by rd-devvv** [comment]
Hi @lauren-mills, I tried that but I also ask for confirmation after user clicks which choice.
in your example if user chose the first(Choose me! (1)) their is a confirmation after it! in your case the confirmation will be are you sure you want to choose myReturnValue1? while i want it to still ask are you sure you want to choose Choose me! (1)? if user clicks yes then the value should be stored as myReturnValue1 and the API will take it as so
in other words I also want to take the title for confirmation and use the value only for API. is this possible?

Thank you

**Comment by anishprasad01** [comment]
Could you clarify what exactly you want to do? I'm not sure I understand.

**Comment by rd-devvv** [comment]
thank you for your help! I managed to implement a workaround for it
